Tests/Tests/CrashLoggingTests.swift

```diff
@@ -1,18 +1,25 @@
 import XCTest
 import Sentry
 
+#if SWIFT_PACKAGE
+@testable import AutomatticTracksModel
+@testable import AutomatticRemoteLogging
+#else
 @testable import AutomatticTracks
+#endif
 
 class CrashLoggingTests: XCTestCase {
 
     private var mockDataProvider = MockCrashLoggingDataProvider()
 
     override func setUp() {
         mockDataProvider.sentryDSN = validDSN
+        UserDefaults.standard.setValue(true, forKey: "force-crash-logging")
     }
 
     override func tearDown() {
         mockDataProvider.reset()
+        UserDefaults.standard.removeObject(forKey: "force-crash-logging")
```

Comment:
Nitpick: we could extract the `"force-crash-logging"` string in an instance `private let` to prevent typos in the future.